src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entries.html.twig

```diff
@@ -20,6 +20,7 @@
 
 {% block content %}
     {% set listMode = app.user.config.listMode %}
+    {% set displayArchivedEntriesRoutes = ['tag_entries', 'search', 'all'] %}
```

Comment:
The variable should have a better name like `entriesWithArchivedClassRoutes` or something similar.
Also, I'm not a huge fan of putting `search` here, we should limit this to searches on all route (`currentRoute=all`)

Reply:
I changed the name. 

About `search`, I added it because I think it's better to have consistency.